docs/modules/ROOT/pages/erc721.adoc

```diff
@@ -141,25 +140,11 @@ mod MyNFT {
         let name = "MyNFT";
         let symbol = "NFT";
         let token_id = 1;
-        let token_uri = "NFT_URI";
+        let base_uri = "https://api.example.com/v1/";
 
         self.erc721.initializer(name, symbol);
-        self._mint_with_uri(recipient, token_id, token_uri);
-    }
-
-    #[generate_trait]
-    impl InternalImpl of InternalTrait {
-        fn _mint_with_uri(
-            ref self: ContractState,
-            recipient: ContractAddress,
-            token_id: u256,
-            token_uri: ByteArray
-        ) {
-            // Initialize the ERC721 storage
-            self.erc721._mint(recipient, token_id);
-            // Mint the NFT to recipient and set the token's URI
-            self.erc721._set_token_uri(token_id, token_uri);
-        }
+        self._mint(recipient, token_id);
+        self._set_base_uri(base_uri);
```

Comment:
```suggestion
        self._set_base_uri(base_uri);
        self._mint(recipient, token_id);
```
very small suggestion, i think the uri is more about setting the contract up, and the minting just a chained operation. this would also be consistent with the preset